doc/known_issues.rst

```diff
@@ -31,13 +31,21 @@ KRKNWK-20869: Intermittent downlink message reception issues with multilink conf
   When multiple transport types are enabled (multilink), downlink messages from the cloud may not be received by the device.
   Sometimes messages arrive via a different transport than expected. Uplink messages work as expected.
 
-  
   **Affected platforms:** nRF52840, nRF54L15 with LR1110 shield
   
   **Note:** The exact set of affected platforms that show this behavior is not constant, and reproduction of this issue is not consistent.
   
   **Impact:** Moderate - affects message reliability in multilink scenarios
 
+
+KRKNWK-20851: can not initialize location when lora or fsk transport is initialized in sidewalk
+
+  **Affected platforms:** nRF52840, nRF54L15 with LR1110 shield
+  
+  **Note:** The reproduction of this issue is not consistent.
+  
+  **Impact:** Moderate - rear device freezes at unsuccessful location initialization, usually restart helps.
```

Comment:
Is this correct? I believe the proper workaround is to keep the proper initialization order?